mmtk.h

```diff
@@ -104,6 +106,10 @@ extern void mmtk_harness_end(void *tls);
 extern void mmtk_register_finalizable(void *reff);
 extern void* mmtk_poll_finalizable(bool include_live);
 
+void rb_mmtk_pre_process_opts(int argc, char **argv);
+void rb_mmtk_post_process_opts(char *arg);
+void rb_mmtk_post_process_opts_finish(void);
+
```

Comment:
The header `mmtk.h` is intended to declare things that are defined in Rust.  But these functions are defined in `gc.c`.  Ideally, they should be declared in `gc.h`.  If possible, we should move these declarations to `gc.h`, unless there are technical difficulties (such as cyclic `include`s) that prevent us from doing so.